src/file_io.py

```diff
@@ -18,18 +18,18 @@
 import numpy
 
 
-def save_item(file, item):
+def save_item(file, item_):
```

Comment:
Just as a general point, the trailing underscore is to distinguish it from item methods in e.g. dictionaries?

Reply:
A trailing underscore is a python convention to distinguish between definitions in a higher scope.

Here, item is already defined, hence appending it.